lambda_function.py

```diff
@@ -0,0 +1,79 @@
+import os
+import re
+
+from nypl_py_utils.functions.log_helper import create_log
+from nypl_py_utils.functions.config_helper import load_env_file
+from record_processor import RecordProcessor
+
+
+def lambda_handler(event, context):
+    logger = create_log("lambda_function")
+    load_env_file(os.environ["ENVIRONMENT"], "config/{}.yaml")
+
+    logger.info("Starting event processing...")
+
+    if event is None:
+        logger.error("Event is undefined.")
+        # TODO: raise exception here?
```

Comment:
agreed -- raise an exception if this happens, although I don't think it ever should